saltproc/saltproc.py

```diff
@@ -205,53 +422,71 @@ def read_res(self, moment):
         [mean_keff, uncertainty_keff]
         """
         res_filename = os.path.join(self.input_file + "_res.m")
-        res = serpent.parse_res(res_filename)
-        keff_analytical = res['IMP_KEFF']
-        return keff_analytical[moment]
-
-    def read_bumat(self, file_name, moment):
-        """ Reads the SERPENT .bumat file
+        count = 0
+        with open(res_filename, 'r') as f:
```

Comment:
I suspect all of these line-by-line file reading pieces are really going to slow down saltproc.